### ISSUE TITLE
[5.x]: Error message logging not working

### ISSUE THREAD

**Issue by bencroker** [craftcms/cms#14863]
### What happened?

Calling `Craft::error()` appears to have no effect in Craft 5.0.5.

### Steps to reproduce

1. Enable `devMode`.
2. In a custom module, call `Craft::error('Test error')` and `Craft::info('Test info')` in the `init()` method.
3. Visit the site.
4. Open the `web-****-**-**.log` file.

### Expected behavior

Both test messages should be written to the log file.

```
[web.ERROR] [application] Test error
[web.INFO] [application] Test info
```

### Actual behaviour

Only the info message is written to the log file.

```
[web.INFO] [application] Test info
```

### Craft CMS version

5.0.5

**Comment by MatthiasBrodelet** [comment]
I have been breaking my head over this issue.
The weird part is that Sentry is able to catch these errors even though they are not visible in the logs.

**Comment by timkelty** [comment]
Confirmed the described behavior.

Errors are also missing with Dev Mode off, but warnings are logged.
Most likely related to the Monolog update that came with Craft 5.

Working on it now!

**Comment by bencroker** [comment]
Thanks, Tim!

**Comment by brandonkelly** [comment]
Fixed in 5.0.6! Thanks again.
